### ISSUE TITLE
Add an option to timestamp filename

### ISSUE THREAD

**Issue by ockam** [jimbojsb/craft-db-snapshot#1]
Thanks for this nice package!

Would it be possible to add an option to timestamp the snapshot filename?

Either through a checkbox where the plugin would manage the timestamp internally
or
through twig expressions on the filename config, as in: `db_snapshot_{now|date('Y')}_{now|date('m')}.sql`

**Comment by ockam** [comment]
Found this method that could help parse the string if you choose option 2:

https://docs.craftcms.com/api/v3/craft-web-view.html#method-renderstring

**Comment by ockam** [comment]
After further digging, in the console controller (at line 48), you could replace
```
        $tmpFile = "$tmpPath/$settings->filename";
        $volumeFile = $settings->filename;
```
with
```
        $view = Craft::$app->getView();
        $filename = $view->renderString($settings->filename);

        $tmpFile = "$tmpPath/$filename";
        $volumeFile = $filename;
```
This would correctly parse the following: `db_snapshot_{{now|date("Ymd-His")}}.sql`

**Comment by jimbojsb** [comment]
That's pretty easy to implement. Might try to cook it up this weekend. I'll also accept a PR for this if you want to take a crack at it.

**Comment by jimbojsb** [comment]
feature is available in 1.1.0
